cli.js

```diff
@@ -44,6 +45,14 @@ if (semver.lt(process.version, '5.0.0')) {
   process.exit(1);
 }
 
+if (typeof cli.flags.mobile === 'undefined') {
```

Comment:
Putting in the runner doesn't move it any AFAICT; we'd still have to default it as `true` for any `undefined`s, but we'd now have to explicitly make the extension set the values to `false` for the initial run.